packages/generator-common/src/sdk-metadata/sdk-metadata.ts

```diff
@@ -56,22 +56,32 @@ export function getSdkMetadataClient(
 const sdkMetadataHeaderIntroText =
   'The SAP Cloud SDK is a versatile set of libraries and tools for developers to build applications in a cloud-native way and host them on the SAP Business Technology Platform or other runtimes.';
 
-const ServiceStatusValues: Record<ServiceStatus['status'], ServiceStatus> = {
+export const ServiceStatusValues: Record<
+  ServiceStatus['status'],
+  ServiceStatus
+> = {
   certified: {
     status: 'certified',
-    statusText: 'A pre-generated API client exists.',
-    gettingStartedText:
-      'For this API you have two options to get a typed client. Either you download the pregenerated client from the repository or you generate the client on your own.'
+    statusText: 'API Client available for download',
+    statusLongText:
+      'You can download our pre-generated API client for this API.'
```

Comment:
```suggestion
      'You can download our pregenerated API client for this API.'
```
I guess we agree to use this without hyphen.